src/frontend/components/StudentLiveWrapper/StudentLiveLayout/index.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { StudentLiveLayout } from '.';
+import { imageSnapshot } from 'utils/tests/imageSnapshot';
+
+const ActionsElement = <p>actions element</p>;
+const LiveTitleElement = <p>live title element</p>;
+const MainCompo = <p>main component</p>;
+const PanelCompo = <p>panel component</p>;
+
+describe('<StudentLiveLayout />', () => {
+  it('render components with panel', async () => {
+    render(
+      <StudentLiveLayout
+        actionsElement={ActionsElement}
+        isPanelOpen={true}
+        liveTitleElement={LiveTitleElement}
+        mainElement={MainCompo}
+        sideElement={PanelCompo}
+      />,
+    );
+
+    screen.getByText('actions element');
+    screen.getByText('live title element');
+    screen.getByText('main component');
+    screen.getByText('panel component');
+
+    expect(screen.getByTestId('panel-id')).toBeVisible();
+
+    await imageSnapshot();
+  });
+
+  it('does not render panel when is isPanelOpen is false', async () => {
+    render(
+      <StudentLiveLayout
+        actionsElement={ActionsElement}
+        isPanelOpen={false}
+        liveTitleElement={LiveTitleElement}
+        mainElement={MainCompo}
+        sideElement={PanelCompo}
+      />,
+    );
+
+    screen.getByText('actions element');
+    screen.getByText('live title element');
+    screen.getByText('main component');
+    screen.getByText('panel component');
+
+    expect(screen.getByTestId('panel-id')).not.toBeVisible();
```

Comment:
Looking at the image snapshot, the panel seems visible.

Reply:
yep, this issue with jsdom and css.
hidden react attribute add a classe setting visibility to none !important yet this attribute is not detected in the style in jsdom.
I tested it with `toHaveStyle ` and even `getComputedStyle` and none worked (which make sense since I think `toHaveStyle` uses `getComputedStyle`)

Reply:
snapshot is mainly usefull to see live bar style since this is set by this component